superset/db_engine_specs.py

```diff
@@ -868,16 +868,17 @@ def get_column_names(filepath):
             secure_filename(form.csv_file.data.filename)
         column_names = get_column_names(upload_path)
         schema_definition = ', '.join(
-            [s + ' STRING ' for s in column_names])
+            [s.decode('utf-8') + ' STRING ' for s in column_names])
```

Comment:
Why do we need to use `decode` here?

Reply:
You were right. It is no longer needed after using unicodecsv. Thanks!